### PR TITLE
Fix typo in documentation

### PR DESCRIPTION
**Pull request by hikhvar**
**What this PR does / why we need it**:
Fix two typos in the documentation

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: docs/using-octavia-ingress-controller.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ kubectl apply -f /etc/kubernetes/octavia-ingress-controller/config.yaml
 ### Deploy octavia-ingress-controller
 
 ```shell
-image="docker.io/k8scloudprovider/octavia-ingress-controller:lastest"
+image="docker.io/k8scloudprovider/octavia-ingress-controller:latest"
```
Comment (lingxiankong) [inline docs/using-octavia-ingress-controller.md]: nice catch :-)

**Comment by hikhvar** [comment]
Rebased the branch. Please take another look.

File: docs/using-octavia-ingress-controller.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ After creating a Kubernetes cluster in Magnum, the most common way to expose the
...
 
-The octavia-ingress-controller could solve all the above problems in the OpenStack environment by creating a single load balancer for multiple [NodePort](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport) type services in an Ingress. In order to use the octavia-ingress-controller in Kubernetes cluster, set the annotation `kubernetes.io/ingress.class` in the `metadata` section of the Ingress resource as shown below:
+The octavia-ingress-controller could solve all the above problems in the OpenStack environment by creating a single load balancer for multiple [NodePort](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport) type services in an Ingress. In order to use the octavia-ingress-controller in Kubernetes cluster, set the annotation `kubernetes.io/ingress.class` in the `metadata` section of the Ingress resource as shown below:
```
Comment (lingxiankong) [inline docs/using-octavia-ingress-controller.md]: I didn't find any differences between those two lines?

Comment (hikhvar) [reply docs/using-octavia-ingress-controller.md]: I removed a ^H between Open and Stack. That was rendered ugly in the github view of the markdown.

Comment (lingxiankong) [reply docs/using-octavia-ingress-controller.md]: hah, ok, thanks!

**Pull request merged**